### PR TITLE
chore: pass CI

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Let's see how many of these we can fix.

### CODE REVIEW

**Review by ShaunSHamilton — APPROVED** [review]
Thanks, for this 👍

**Comment by raisedadead** [comment]
Nice work @nhcarrigan

File: client/src/assets/icons/spacer.tsx [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-// eslint-disable-next-line
-function Spacer(props: any): JSX.Element {
+interface SpacerPropTypes {
+  style: Record<string, unknown>;
+}
+
+function Spacer(props: SpacerPropTypes): JSX.Element {
   const { t } = useTranslation();
 
+  props.style = { paddingTop: '5', ...props.style };
```
Comment (ojeytonwilliams) [inline client/src/assets/icons/spacer.tsx:11]: Doesn't this change how it's displayed? It's gone from paddingTop to style.paddingTop

Comment (ShaunSHamilton) [reply client/src/assets/icons/spacer.tsx:11]: My understanding is `paddingTop` is not a prop on `svg`, thus needs to be in the `style` attribute.

Comment (ojeytonwilliams) [reply client/src/assets/icons/spacer.tsx:11]: That sounds right, but doesn't that mean these will be padded now?

Comment (ShaunSHamilton) [reply client/src/assets/icons/spacer.tsx:11]: Ah... I will find out.

Comment (nhcarrigan) [reply client/src/assets/icons/spacer.tsx:11]: They did already have a `paddingTop` attribute, I just moved it.

Comment (ojeytonwilliams) [reply client/src/assets/icons/spacer.tsx:11]: Oh, not to worry then.  Thanks, Nick.

**Pull request merged**